span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -123,6 +136,16 @@ public RawSpan convert(String tenantId, Span jaegerSpan) throws Exception {
       if (LOG.isDebugEnabled()) {
         logSpanConversion(jaegerSpan, rawSpan);
       }
+
+      // register and update timer per tenant
+      tenantToDelayInSpanProcessedTimer
+          .computeIfAbsent(
+              tenantId,
+              tenant ->
+                  PlatformMetricsRegistry.registerTimer(
+                      DELAY_IN_SPAN_PROCESSED_TIME_METRIC, Map.of("tenantId", tenant), true))
+          .record(spanProcessedTime - spanStartTime, TimeUnit.MILLISECONDS);
```

Comment:
corner case, but clock skews could result in spanStartTime being more than spanProcessedTime. Should we first check if difference is not negative?

Reply:
Same question I had - https://github.com/hypertrace/hypertrace-ingester/pull/304/#discussion_r812662035

Let me do that.

Reply:
@satish-mittal I have wrapped the difference around `math.abs` as if there is a very large skew we would like to know, right?